browser/src/control/Control.GroupBase.ts

```diff
@@ -83,6 +83,16 @@ export abstract class GroupBase extends app.definitions.canvasSectionObject {
 		L.DomUtil.remove(elem);
 	}
 
+	private _groupBaseColors(): void {
```

Comment:
This function seems to be a duplicate of similar one in the corner group.
Can we share it?

example as static method of GroupBase class. So we can use it as `GroupBase.getColors()`

Reply:
great! thanks